### ISSUE TITLE
Redstone augment not behaving properly

### ISSUE THREAD

**Issue by jinkhya** [desht/ModularRouters#145]
### Minecraft Version
1.16.5
### Forge Version
36.2.2
### Mod Version
7.5.1-69
### Describe your problem, including steps to reproduce it
Unless I set this up wrongly, something's wrong with the redstone augment. To my understanding I can put the item router on "redstone mode: high" and then put a redstone module in a puller module with "redstone mode: always" which would mean all the modules in that item router won't work until the item router receives redstone except for the puller module which should work at all times. That doesn't seem to work, the puller doesn't work either if no redstone power is given to the item router. 
Thanks in advance.
Jin

**Comment by desht** [comment]
Yeah you're not wrong.  The router actually doesn't bother to run (or even check) any modules if it doesn't run itself.

It could be debated whether or not that's the correct behaviour, of course.  The workaround would be to keep the router in "always" mode and add augments to the other modules (the ones which should only run on a high signal).  This does potentially require more augments, of course.

I'll have a think about this and decide on the best way forward.  Changing the router behaviour is possible, but it's a significant internal rework of the way modules are executed right now, and could possibly lead to some existing builds breaking...

**Comment by jinkhya** [comment]
Workaround worked great ! I assumed it wouldn't work the other way either. Thank you :)
Good luck on the decision about your best next plan of action.
